include/ensmallen_bits/nsga2/nsga2_impl.hpp

```diff
@@ -365,6 +377,8 @@ inline void NSGA2::FastNonDominatedSort(
     i++;
     fronts.push_back(nextFront);
   }
+  //Remove the empty final set
```

Comment:
```suggestion
  // Remove the empty final set.
```

Another minor style fix, not sure I mentioned this at some point, but ensmallen follows https://github.com/mlpack/mlpack/wiki/DesignGuidelines.